torch_glow/src/PyTorchModelLoader.cpp

```diff
@@ -1209,6 +1224,35 @@ llvm::Error PyTorchModelLoader::loadAvgPool2d(const torch::jit::Node *ptNode) {
   return addGlowNodeValue(outputs[0], output);
 }
 
+llvm::Error PyTorchModelLoader::loadClamp(const torch::jit::Node *ptNode) {
+  auto inputs = ptNode->inputs();
+  auto outputs = ptNode->outputs();
+  RETURN_IF_ERR(checkInputAndOutputSizes(inputs, 3, outputs, 1));
+
+  glow::NodeValue input;
+  ASSIGN_VALUE_OR_RETURN_ERR(input,
+                             getGlowNodeValue(inputs[ClampInputs::input]));
+
+  auto minHandle = glow::Handle<float>::createInvalidHandle();
+  ASSIGN_VALUE_OR_RETURN_ERR(
+      minHandle, getGlowConstantHandle<float>(inputs[ClampInputs::min]));
+  RETURN_ERR_IF_NOT(
+      minHandle.size() == 1,
+      glow::strFormat("min size must be 1, got %lu", minHandle.size()));
+  auto min = minHandle.raw(0);
+
+  auto maxHandle = glow::Handle<float>::createInvalidHandle();
+  ASSIGN_VALUE_OR_RETURN_ERR(
+      maxHandle, getGlowConstantHandle<float>(inputs[ClampInputs::max]));
+  RETURN_ERR_IF_NOT(
+      maxHandle.size() == 1,
+      glow::strFormat("max size must be 1, got %lu", maxHandle.size()));
+  auto max = maxHandle.raw(0);
+
+  auto output = F_.createClip("clip", input, min, max);
```

Comment:
Note I'm assuming min/max are constant, which PyTorch decidedly does not guarantee.  This seems like one more instance of the general caching problem so I just figured I'd not worry about it for the time being...

Reply:
Yeah, do not worry about it here.